### PR TITLE
fix:locale storage dropdown bug  (#500)

### PR DESCRIPTION
**Pull request by kujo205**

### CODE REVIEW

File: src/components/common/ui/form/dropdown/Dropdown.tsx [hunk 1]
```diff
@@ -89,10 +91,14 @@ export const Dropdown: FC<DropdownProps> = ({
           }}
           fullWidth
           disablePortal
-          onChange={handleChange}
           blurOnSelect={true}
           options={options}
-          isOptionEqualToValue={(option, value) => option.value === value.value}
+          isOptionEqualToValue={(option, value) => {
```
[the remark this answers is not in the document]
Comment (kujo205) [reply src/components/common/ui/form/dropdown/Dropdown.tsx:96]: Well initially i introduced this code to fix the following warning
image
But now after I deleted if clause this mistake appears once again.
Although it disappeared in the prod

[the remark this answers is not in the document]
Comment (andrewvitrenko) [reply src/components/common/ui/form/dropdown/Dropdown.tsx:96]: okay, but leave it in one line as it was before `(option, value) => option.value === value.value`. and leave comment above this string `TODO: review option structure`

**Review by andrewvitrenko — APPROVED** [review]

File: src/components/common/ui/form/dropdown/Dropdown.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (kujo205) [reply src/components/common/ui/form/dropdown/Dropdown.tsx:96]: I cannot push changes with a todo 😁

[the remark this answers is not in the document]
Comment (kujo205) [reply src/components/common/ui/form/dropdown/Dropdown.tsx]: btw, what this to-does are meant for in a real-life company. Do teamleads assign them or do programmers write them for themsels when  they're given tasks

**Pull request merged**